docs/resources/node_template.md

```diff
@@ -217,6 +217,7 @@ The following attributes are exported:
 
 * `api_token` - (Required/Sensitive) Hetzner Cloud project API token (string)
 * `image` - (Optional) Hetzner Cloud server image. Default `ubuntu-18.04` (string)
+* `server_labels` - (Optional) Map of labels which will be assigned to the server. This argument is only available on [Hetzner Docker Node Driver:v3.6.0](https://github.com/JonasProgrammer/docker-machine-driver-hetzner/releases/tag/3.6.0) and above (map)
```

Comment:
Could you also please, update [hetzner example url](https://github.com/rancher/terraform-provider-rancher2/blame/master/docs/resources/node_template.md#L67) setting the proper version to be able to use this argument??